Monika After Story/game/script-moods.rpy

```diff
@@ -1024,3 +1024,78 @@ label mas_mood_guilty:
     m 1ekbsu "I love you.{w=0.2} Don't be so hard on yourself, okay?"
     $ mas_moni_idle_disp.force_by_code("1ekbla", duration=5, skip_dissolve=True)
     return "love"
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_cozy",
+            prompt="...cozy.",
+            category=[store.mas_moods.TYPE_GOOD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_cozy:
+    m 1eua "Oh?{w=0.3} What makes you feel cozy?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "Oh? What makes you feel cozy?{fast}"
+
+        "Being with you.":
+            m 1ekbsb "Aww, [player].{w=0.3} I am so happy I'm making you feel that way."
+            m 1eubsb "I love spending time with you too."
+            m 1dkbsa "You being cozy makes me feel cozy as well..."
+            $ _now = datetime.datetime.now().time()
+            $ is_night = mas_isSStoMN(_now) or mas_isMNtoSR(_now)
+            if mas_isMoniEnamored(higher=True) or mas_isMoniAff(higher=True) and is_night:
+                call monika_holdrequest
```

Comment:
We definitely don't want to call this label here - it's a random event that may be locked when the player uses this mood. That also means that you skip the conditional check for that event. If you just want a hug here, you can use the relevant labels here to setup the holdme request instead of calling the event label.